apmpackage/docs/README.template.md

```diff
@@ -1,25 +1,63 @@
 # APM Integration
 
-The APM integration installs templates and pipelines for APM data.
-If a policy contains an `apm` input, any Elastic Agent(s) set up with that policy will run an APM Server binary, and bind to `localhost:8200`.
+The APM integration installs Elasticsearch templates and Ingest Node pipelines for APM data.
+
+### How to use this integration
+
+When you add an APM integration to a policy, that policy will contain an `apm` input.
+If a policy contains an `apm` input, any Elastic Agent(s) set up with that policy will run locally an APM Server binary.
 You must configure your APM Agents to communicate with that APM Server.
 
 If you have RUM enabled, you must run APM Server centrally. Otherwise, you can run it at the edge machines.
 To do so, download and enroll an Elastic Agent in the same machines where your instrumented services run.
 
+If you want to change the default APM Server configuration, you need to edit the `elastic-agent.yml` policy file manually.
```

Comment:
In the Fleet UI we only have one setting for enable/disable RUM for now, so this is the only way to configure other stuff. I believe other integrations have a similar note somewhere. We can move it elsewhere so that doesn't feel much "encouraging" but I'll leave that to the docs team.

Reply:
Hmm, okay - thi is fine for now, we can revisit it later.